bool valid
builtin_interfaces/Time latest_timestamp
uint8 arming_state
uint8 nav_state
bool failsafe
bool pre_flight_checks_pass
